Dockerfile.txt

```diff
@@ -412,7 +412,8 @@ RUN ./install_bioc.r \
 
 # Package for removing unwanted variation from RNA-Seq data
 RUN ./install_bioc.r \
-    RUVSeq
```

Comment:
I think this is why you have a merge conflict - can just carry over Dockerfile from dev, as this one has `.txt`, which it should not

Reply:
Oh I see - you did pull over, but can you remove the `.txt` extension?